DTS-STN/canadian-dental-care-plan
# Placeholder Dependent Status Check added.

### Description

This is placeholder based on the existing status check.

### Related Azure Boards Work Items

[AB#3366](https://dev.azure.com/DTS-STN/Canada%20Dental%20Care%20Plan/_workitems/edit/3366)

### Screenshots (if applicable)


![Screenshot from 2024-05-10 10-56-58](https://github.com/DTS-STN/canadian-dental-care-plan/assets/154452024/956cd93c-978f-4410-a264-b93aa95c4b5e)

![Screenshot from 2024-05-10 10-56-19](https://github.com/DTS-STN/canadian-dental-care-plan/assets/154452024/923da419-d214-48e7-a58a-be7480459e26)



### Checklist


### Test Instructions


1. Include `dependent-status-checker` in the list of `ENABLED_FEATURES` in the .env file
2. Open the protected home page
3. Confirm the presence of the Dependent Status Checker card
4. Click the Dependent Status Checker card
5. Confirm that the Dependent Status Checker placeholder is displayed
